server/src/repository/mod.rs

```diff
@@ -13,9 +13,11 @@ use crate::config::CONFIG;
 use crate::package::Package;
 
 mod manage;
+pub mod crypto;
 
 const REPO_DIR: &str = "repository";
 const REPO_SERENE: &str = "bases.json";
+const PRIV_KEY_FILE: &str = "sign-private-key.sec.asc";
```

Comment:
I haven't seen the `.sec` extension anywhere on the internet, seems odd to me. I would opt for `signing_key.asc`, being snake_case to be consistent with authorized_secrets